pygus/gus/models.py

```diff
@@ -114,23 +124,40 @@ def __init__(
             # Place trees on the plot based on actual physical positioning
             x = row.xpos
             y = row.ypos
+            logging.debug("Placing agent {} at ({},{})".format(index, x, y))
             self.grid.place_agent(a, (x, y))
 
         # This variable below works as an indexer while adding new trees to the population during the run time.
         self.current_id = max(population.id)
 
+        ALIVE_STATE = ["excellent", "good", "fair", "poor", "critical", "dying"]
         # Collecting model and agent level data
         self.datacollector = DataCollector(
             model_reporters={
                 "Storage": lambda m: self.aggregate(m, "carbon_storage"),
                 "Seq": lambda m: self.aggregate(m, "annual_gross_carbon_sequestration"),
                 # "Sequestrated": self.aggregate_sequestration,
+                # Avg sequestered carbon per tree is annual carbon divided by number of living trees
+                "Avg_Seq": lambda m: self.aggregate(
+                    m, "annual_gross_carbon_sequestration"
+                )
+                / (
+                    self.count(
+                        m,
+                        "condition",
+                        lambda x: x in ALIVE_STATE,
+                    )
+                ),
                 "Released": self.compute_current_carbon_release,
-                "Alive": lambda m: self.count(
+                # Avg released carbon per year is annual carbon release divided by number of living trees
+                "Avg_Rel": lambda m: self.compute_current_carbon_release(m)
+                / self.count(
```

Comment:
Avg Seq and Rel have been added here that take into account the annual alive tree population